src/prime_rl/trainer/config.py

```diff
@@ -219,7 +219,10 @@ class TrainerConfig(BaseSettings):
     @model_validator(mode="after")
     def auto_setup_bench(self):
         if self.bench:
-            self.max_steps = 6  # 1 Warmup + 5 Benchmark
+            self.max_steps = 4  # 1 Warmup + 3 Benchmark
             if not self.data.fake:
                 self.data.fake = FakeDataLoaderConfig()
+
+            if self.monitor.wandb:
+                self.monitor.wandb.project = f"{self.monitor.wandb.project}-bench"
```

Comment:
not sure about this one seems to intrusive

either we use a fix bench project and the user is in charge to pick to add -bench